pvlib/spectrum/__init__.py

```diff
@@ -1,14 +1,18 @@
 from pvlib.spectrum.spectrl2 import spectrl2  # noqa: F401
 from pvlib.spectrum.mismatch import (  # noqa: F401
     calc_spectral_mismatch_field,
-    get_am15g,
-    get_reference_spectra,
-    get_example_spectral_response,
     spectral_factor_caballero,
     spectral_factor_firstsolar,
     spectral_factor_sapm,
     spectral_factor_pvspec,
-    spectral_factor_jrc,
+    spectral_factor_jrc
```

Comment:
```suggestion
    spectral_factor_jrc,
```

Nitpick: it's nice to retain the trailing comma in a multi-line list like this.  In the future, if we want to add a new element in the list, then the diff is only one line (the new element) rather than two (the new element, plus editing the previous line to add the comma).  See https://www.pythonmorsels.com/trailing-commas-in-python/

Reply:
Thanks for picking up on that. I have made the change not only to this multiline list but the other two lists fas well.